src/main/java/com/upsolver/datasources/jdbc/JDBCDataSource.java

```diff
@@ -144,6 +159,7 @@ private TableInfo loadTableInfo(DatabaseMetaData metadata, String schemaPattern,
     public List<PropertyDescription> getPropertyDescriptions() {
         ArrayList<PropertyDescription> result = new ArrayList<>();
         result.add(new SimplePropertyDescription(connectionStringProp, "The connection string that will be used to connect to the database", false));
+        result.add(new SimplePropertyDescription(connectionPropertiesProp, "Extra connection properties that will be used to connect to the database", true));
```

Comment:
You need to add some boolean to this to allow 'textArea' and then use it in the generation of ExternalInputRequests